# CAN Bus Message

Header header
uint8 ID
uint8 PreIndex
uint16 Index
uint8 SubIndex
int32 Data